assignments/array-methods.js

```diff
@@ -58,28 +58,74 @@ const runners = [
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs. Combine both the first and last names and populate a new array called `fullNames`. This array will contain just strings.
 let fullNames = [];
+runners.forEach(names => {
+  fullNames.push(`${names.first_name} ${names.last_name}`)
+});
 console.log(fullNames);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runners' first names in uppercase because the director BECAME DRUNK WITH POWER. Populate an array called `firstNamesAllCaps`. This array will contain just strings.
 let firstNamesAllCaps = [];
+
+function allCaps(fNames) {
+  return fNames.first_name.toUpperCase();
+}
+firstNamesAllCaps = runners.map(allCaps);
+
+//Short way
+// firstNamesAllCaps = runners.map( fNames => fNames.first_name.toUpperCase() );
+
```

Comment:
The commented code is what you should have use and implemented. That is how to use map